recipes/soundtouch/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class SoundTouchConan(ConanFile):
+    name = "soundtouch"
+    description = "an open-source audio processing library that allows changing the sound tempo, pitch and playback rate parameters independently"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://codeberg.org/soundtouch/soundtouch"
+    topics = ("audio", "processing", "tempo", "pitch", "playback")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "integer_samples": [True, False],
+        "with_openmp": [True, False],
+        "with_dll": [True, False],
+        "with_util": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "integer_samples": False,
+        "with_openmp": False,
+        "with_dll": False,
+        "with_util": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["INTEGER_SAMPLES"] = self.options.integer_samples
+        tc.variables["SOUNDTOUCH_DLL"] = self.options.with_dll
+        tc.variables["SOUNDSTRETCH"] = self.options.with_util
+        tc.variables["OPENMP"] = self.options.with_openmp
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="COPYING.TXT", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "SoundTouch")
+
+        self.cpp_info.components["_soundtouch"].set_property("cmake_target_name", "SoundTouch::SoundTouch")
+        self.cpp_info.components["_soundtouch"].set_property("pkg_config_name", "SoundTouch")
```

Comment:
```suggestion
        self.cpp_info.components["_soundtouch"].set_property("pkg_config_name", "soundtouch")
```
I built without Conan and cmake install provided soundtouch.pc

```
% cmake --build . --target install 
[ 78%] Built target SoundTouch
[100%] Built target soundstretch
Install the project...
-- Install configuration: "Release"
-- Installing: /tmp/install/include/soundtouch/BPMDetect.h
-- Installing: /tmp/install/include/soundtouch/FIFOSampleBuffer.h
-- Installing: /tmp/install/include/soundtouch/FIFOSamplePipe.h
-- Installing: /tmp/install/include/soundtouch/STTypes.h
-- Installing: /tmp/install/include/soundtouch/SoundTouch.h
-- Installing: /tmp/install/include/soundtouch/soundtouch_config.h
-- Installing: /tmp/install/lib/libSoundTouch.a
-- Installing: /tmp/install/bin/soundstretch
-- Installing: /tmp/install/lib/pkgconfig/soundtouch.pc
-- Installing: /tmp/install/lib/cmake/SoundTouch/SoundTouchTargets.cmake
-- Installing: /tmp/install/lib/cmake/SoundTouch/SoundTouchTargets-release.cmake
-- Installing: /tmp/install/lib/cmake/SoundTouch/SoundTouchConfig.cmake
-- Installing: /tmp/install/lib/cmake/SoundTouch/SoundTouchConfigVersion.cmake
```